app/layout.tsx

```diff
@@ -1,17 +1,19 @@
 import type { Metadata } from "next";
-import { Geist, Geist_Mono } from "next/font/google";
+import { Share_Tech, Share_Tech_Mono } from "next/font/google";
 import "./globals.css";
 import Navbar from "@components/Navbar";
 import Footer from "@components/Footer";
 
-const geistSans = Geist({
+const geistSans = Share_Tech({
   variable: "--font-geist-sans",
```

Comment:
```suggestion
const shareTechSans = Share_Tech({
  variable: "--font-share-tech-sans",
```
Now that we are using `Share Tech` in place of `Geist` this should be reflected in the variable names